src/components/app/content/index.js

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './index.css';
-import Card from "./card";
+import Cards from "./cards";
 
 function Content() {
+    const [readOnly, setReadOnly] = useState(false)
+    const [cards, setCards] = useState([
+        {id: 1, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 2, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 3, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 4, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 5, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 6, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 7, header: 'Caption', body: 'Text...', checked: false, editable: false},
+        {id: 8, header: 'Caption', body: 'Text...', checked: false, editable: false},
```

Comment:
editable is something more related to Card component for now. Better move it to Card component state.